tests/functional/m/method_hidden.py

```diff
@@ -1,7 +1,10 @@
 # pylint: disable=too-few-public-methods,missing-docstring
 # pylint: disable=unused-private-member
+# pylint: disable=import-error
```

Comment:
Cosmetic suggestion: I'd suggest disabling this on line 6 only, not here for the entire file.